README.md

```diff
@@ -20,38 +53,23 @@ Variable | Default | Meaning
   hosts: galaxy
   become: true
   vars:
-    galaxy_subsite_base_domain: usegalaxy.aq
-    galaxy_subsites:
-      - name: virus # Accessible at virus.usegalaxy.aq
-        brand: Viruses
-        iframe: "https://galaxyproject.aq/viruses.html"
-        tool_sections: # The tool section ids to display, generic tools will always be displayed and the list of these can be found in `defaults/main.yml`
+    subdomains_themes_conf_path: files/galaxy/themes.yml
+    subdomains_themes_files: "{{ lookup('fileglob', 'files/galaxy/themes/*.yml', wantlist=True) }}"
+    subdomains_base_indentation: "    "
```

Comment:
I do not have good feelings about this line 

if this role runs after galaxy, I have a very strong preference for instead copying the galaxy_config, adding our variables, and writing that out separately that admins should point their systemd units to. thats kind of gross but way, way, way less error prone than hoping the indentation matches up properly.

if this role runs before galaxy_config, I think you should be able to just ammend the configuration and let the galaxy role serialize it properly. This would be my preference but I guess you need to copy static files (is symlinking an option?)

modifying galaxy.yml in this way will present changes every single playbook run.

Reply:
I don't get the first part with the systemd units sorry